File: cli/tests/projects/test-frameworks/failing/go/failing_test.go
package failing

import "testing"

func TestShouldFailAssertion(t *testing.T) {
	// This test should fail
	result := 1 + 1
	expected := 3
	if result != expected {
		t.Errorf("Expected %d but got %d", expected, result)
	}
}

func TestShouldPass(t *testing.T) {
	// This test should pass
	result := 2 + 2
	expected := 4
	if result != expected {
		t.Errorf("Expected %d but got %d", expected, result)
	}
}

func TestShouldFailString(t *testing.T) {
	// This test should fail
	result := "hello"
	expected := "world"
	if result != expected {
		t.Errorf("Expected %q but got %q", expected, result)
	}
}

File: cli/tests/projects/test-frameworks/failing/node/tests/failing.test.js
// Intentionally failing tests to verify error handling

describe('Failing Tests', () => {
  test('this test should fail - assertion error', () => {
    expect(1 + 1).toBe(3); // Wrong expectation
  });

  test('this test should pass', () => {
    expect(2 + 2).toBe(4);
  });

  test('this test should fail - string mismatch', () => {
    expect('hello').toBe('world');
  });
});
